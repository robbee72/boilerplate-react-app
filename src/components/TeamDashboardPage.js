import React from 'react';
import TeamList from './TeamList';
import TeamListFilters from './TeamListFilters';
import TeamsSummary from './TeamsSummary';

const TeamDashboardPage = () => (
  <div>
    <TeamsSummary />
    <TeamListFilters />
    <TeamList />
  </div>
);

export default TeamDashboardPage;
