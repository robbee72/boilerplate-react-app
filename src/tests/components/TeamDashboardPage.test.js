import React from 'react';
import { shallow } from 'enzyme';
import TeamDashboardPage from '../../components/TeamDashboardPage';

test('should render TeamDashboardPage correctly', () => {
  const wrapper = shallow(<TeamDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
