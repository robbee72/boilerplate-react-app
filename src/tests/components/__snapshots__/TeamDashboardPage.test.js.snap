// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TeamDashboardPage correctly 1`] = `
<div>
  <Connect(TeamsSummary) />
  <Connect(TeamListFilters) />
  <Connect(TeamList) />
</div>
`;
